doc/srcdeps.yaml

```diff
@@ -183,6 +188,62 @@ maven:
     - my-bare-breaker-property
     - my-breaker-property=my-breaker-value
 
+  # `failWithout` - a configuration node to define conditions whose absence will make the outer Maven build fail
```

Comment:
Agree then :) Perhaps the code could be simplified by having only a single property, but if the end user experience is better this way, I'm all for it.